### ISSUE TITLE
Api not exists error

### ISSUE THREAD

**Issue by iiei** [rg3/youtube-dl#8429]
I can't download videos from Naver Vlive
```
youtube-dl -v http://www.vlive.tv/video/5247
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'http://www.vlive.tv/video/5247']
[debug] Encodings: locale cp949, fs mbcs, out cp949, pref cp949
[debug] youtube-dl version 2016.02.04
[debug] Python version 2.7.10 - Windows-7-6.1.7601-SP1
[debug] exe versions: none
[debug] Proxy map: {}
[vlive] 5247: Download video page
[vlive] 5247: Downloading video json
ERROR: GPOP_globalV/vod/5247/playinfo : Api not exists ; please report this issue on https://yt-dl.org/bug . Make sure you are using
the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl w
ith the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "youtube_dl\YoutubeDL.pyo", line 666, in extract_info
  File "youtube_dl\extractor\common.pyo", line 313, in extract
  File "youtube_dl\extractor\vlive.pyo", line 55, in _real_extract
ExtractorError: GPOP_globalV/vod/5247/playinfo : Api not exists (\uc874\uc7ac\ud
558\uc9c0 \uc54a\ub294 API \uc785\ub2c8\ub2e4.); please report this issue on htt
ps://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-d
l -U  to update. Be sure to call youtube-dl with the --verbose flag and include
its complete output.
```

**Comment by EraYaN** [comment]
PR #6621 seems to need an update, it seems. Naver has disabled all downloading though the (previous) API.

The web version has now switched to a Flash Player player, but the underlying stream is still the same, still *.ts parts without any DRM.

**Comment by EraYaN** [comment]
Implemented in #8446, waiting for merge.

**Comment by dstftw** [comment]
This issue has been fixed and fix will be incorporated in the next version of youtube-dl.
